### PR TITLE
added log4j-faq text

### PR DESCRIPTION
**Pull request by browkin**
- changed & added some text in the log4j-faq

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
                         "active": true,
                         "textblock": [
                             "Log4j is a widely used Java-based utility for logging error messages in applications. The Log4j library is widely used in a very broad range of enterprise software and services from many major vendors. A vulnerability related to the Log4j library was discovered recently and allows remote code execution.",
-                            "In response to this recent discovery and in order to minimize potential impacts the project teams of SAP and Deutsche Telekom have joined the worldwide effort and taken action. This includes amongst others a server update to 2.16.0 of the log4j library.",
+                            "In response to this recent discovery and in order to minimize potential impacts the project teams of SAP and Deutsche Telekom have joined the worldwide effort and taken action. This also includes the update to 2.16.0 of the affected log4j library leveraged by the server component. There is no need to take action by the user, both the Corona warning app and the backend systems are safe.",
```
Comment (Ein-Tim) [inline src/data/faq.json:100]: "Corona warning app" -> "Corona-Warn-App"

```suggestion
                            "In response to this recent discovery and in order to minimize potential impacts the project teams of SAP and Deutsche Telekom have joined the worldwide effort and taken action. This also includes the update to 2.16.0 of the affected log4j library leveraged by the server component. There is no need to take action by the user, both the Corona-Warn-App and the backend systems are safe.",
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Please see the in-line suggestion.

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim Thanks

**Review by Ein-Tim — APPROVED** [review]
LGTM now.

**Comment by MikeMcC399** [comment]
It's a bit messy to display the actual search string instead of hiding it in a hyperlink:

https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j

I don't suggest yet another PR for this, but if another one is needed for another reason, then perhaps this could be made prettier?

Also the search string finds **all** PRs for log4j, not just the ones related to the update of the log4j library to 2.16.0.

https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j+2.16.0+in%3Atitle would filter the search better.

**Pull request merged**